### PR TITLE
CPS-198: Replace `crmApi` with `civicaseCrmApi`

### PR DESCRIPTION
**Pull request by deb1990**
## Overview
Replaced `crmApi` with `civicaseCrmApi` throughout the Civicase application.

## Before/After
No visual change occurred.

## Technical Details
In https://github.com/compucorp/uk.co.compucorp.civicase/pull/438, `civicaseCrmApi` was created. Now this has been applied throughout the application.
When `crmApi` is used with an array of api calls, it suppresses any error thrown by the backend. So errors go unnoticed. Now that we have switched to `civicaseCrmApi`, this will be resolved.

### CODE REVIEW

File: ang/civicase/activity/calendar/directives/activities-calendar.directive.js [hunk 1]
```diff
@@ -160,13 +160,13 @@
    * @param {*} $sce sce service
    * @param {*} $scope $scope scope object
    * @param {*} ContactsCache contacts cache service
-   * @param {*} crmApi crm api service
+   * @param {*} civicaseCrmApi crm api service
    * @param {*} formatActivity format activity service
    * @param {*} getActivityFeedUrl get activity feed url service
    * @param {*} ActivityStatusType activity status type service
```
Comment (reneolivo) [inline ang/civicase/activity/calendar/directives/activities-calendar.directive.js:166]: can we fix these parameters?

File: ang/civicase/activity/feed/directives/activity-month-nav.directive.js [hunk 2]
```diff
@@ -15,11 +15,11 @@
...
      *
-     * @param {Object} scope
-     * @param {Object} $el
-     * @param {Object} attr
+     * @param {object} scope scope object
+     * @param {object} $element directives html element
+     * @param {object} attr attributes of the directive
      */
-    function civicaseActivityMonthNavLink (scope, $el, attr) {
+    function civicaseActivityMonthNavLink (scope, $element, attr) {
```
Comment (reneolivo) [inline ang/civicase/activity/feed/directives/activity-month-nav.directive.js:22]: thank you!!!!

File: ang/civicase/case/details/file-tab/directives/file-list.directive.js [hunk 3]
```diff
@@ -31,20 +29,20 @@
     /**
      * Refreshes the UI state after updating the db from the api calls
      *
-     * @params {Array} apiCalls
+     * @param {Array} apiCalls lisy of api calls
```
Comment (reneolivo) [inline ang/civicase/case/details/file-tab/directives/file-list.directive.js:32]: list*

File: ang/civicase/contact/services/contacts-cache.service.js [hunk 4]
```diff
@@ -3,7 +3,14 @@
 
   module.service('ContactsCache', ContactsCache);
 
-  function ContactsCache (crmApi, $q, $rootScope) {
+  /**
+   * Contacts Cashe Service
```
Comment (reneolivo) [inline ang/civicase/contact/services/contacts-cache.service.js:7]: * cache

Comment (deb1990) [reply ang/civicase/contact/services/contacts-cache.service.js:7]: Oops.

**Pull request merged**